qiskit_ibm/api/clients/runtime.py

```diff
@@ -179,7 +179,8 @@ def jobs_get(
             limit: int = None,
             skip: int = None,
             pending: bool = None,
-            program_id: str = None
+            program_id: str = None, 
+            provider: str = None
```

Comment:
@kt474 This needs to be hub, group and project as separate parameters, in all functions.

Reply:
Sure, but I believe the endpoint only works if all three (hub, group, project) inputs are given. If there is a hub parameter given but no group/project, all jobs will be returned instead of jobs from just the given hub - is this intended?

Reply:
User should set all 3, else raise `IBMInputValueError`. See example:
https://github.com/Qiskit-Partners/qiskit-ibm/blob/f89770b962c2b2aabb7ca126d753016795303f88/qiskit_ibm/ibm_provider.py#L689-L693